src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs

```diff
@@ -247,6 +248,7 @@ internal virtual void OnSetNodeDeltaState(DeltaComputeStateEventArgs e)
         /// <param name="verboseLogging">Indicates if detailed descriptions should be logged</param>
         /// <param name="isTestMode">Indicates if current code is running in tests</param>
         /// <param name="fileName">Name of file where the workspace is saved</param>
+        [Obsolete("please use the version with linterManager parameter.")]
```

Comment:
some questions on this:
1. have you verified that all these constructors can be replaced by your new constructor? - What I mean is, do they cover any other use cases?
2. If a file is deserialized with no linterManager - should null be passed?

Reply:
- i would say I've tested this as much as possible, also all different constructors have gotten a new equivalent just with the linterManager param. Before the changes there were 3 ctors, I've add 3 new ones that replicate the old ones but with that new param.
- Essentially the only thing that is serialized from the linter manager is the active linter (name and id) and issue counts (the counts are only for things like GD and player). If there is no active linter when a graph is saved we just serialize `Linting : []`, when deserializing all we are doing is setting the active linter to whatever was serialized (if it's available on the manager). The linter manager is created on the DynamoModel, so if a workspace is deserialized without an active linter value we just don't set any active linter on the LinterManager.